Magento2/Helpers/Commenting/PHPDocFormattingValidator.php

```diff
@@ -123,6 +126,10 @@ public function hasDeprecatedWellFormatted($commentStartPtr, $tokens)
         }
         $seePtr = $this->getTagPosition('@see', $commentStartPtr, $tokens);
         if ($seePtr === -1) {
+            if (stripos($tokens[$deprecatedPtr + 2]['content'], self::REMOVED_IN_VERSION, 0) !== false &&
```

Comment:
Expected:
@deprecated This method will be removed in version 2.x.x without replacement

Actual:
1. removed in versionwithout replacement
2. without replacementremoved in version

Reply:
Thanks for the useful comment! I've done a refactor to use preg_match instead